_episodes/05-lists.md

```diff
@@ -367,18 +367,18 @@ last: 4
 > Use slicing to access only the last four characters of a string or entries of a list.
 >
 > ~~~
-> string_for_slicing = "Observation date: 02-Feb-2013"
-> list_for_slicing = [["fluorine", "F"],
->                     ["chlorine", "Cl"],
->                     ["bromine", "Br"],
->                     ["iodine", "I"],
->                     ["astatine", "At"]]
+> string_for_slicing = 'Observation date: 02-Feb-2013'
+> list_for_slicing = [['fluorine', 'F'],
+>                     ['chlorine', 'Cl'],
+>                     ['bromine', 'Br'],
+>                     ['iodine', 'I'],
+>                     ['astatine', 'At']]
 > ~~~
 > {: .language-python}
 >
 > ~~~
-> "2013"
-> [["chlorine", "Cl"], ["bromine", "Br"], ["iodine", "I"], ["astatine", "At"]]
+> 2013
```

Comment:
I believe this should have single quotes around it:
```suggestion
> '2013'
```

Reply:
I believe this line is the representation of the command output...when I tested this in my particular Jupyter notebook at least, this output from print didn't have quotes around it (it usually wouldn't, would it?)

Reply:
Unless we change the solution section below, output should have single quotes around it because we're not printing here. 
![image](https://user-images.githubusercontent.com/13123663/87971901-b2f6b900-ca8b-11ea-9bfa-57b9b594a91c.png)

Reply:
Ah, that's right, I see. I also remember now when I was running through this I noticed something interesting. It didn't even work for me as given. It might be a Jupyter-related particular, and might work differently from another interpreter. But mine looks like this when I try to output both lists as given in the example:
![image](https://user-images.githubusercontent.com/31045615/87972726-fd2c6a00-ca8c-11ea-88f5-d1874ee1e49c.png)
If each is run separately they'll both output, but called in the same cell only the last one seems to show up (unless I changed it to use print).

Reply:
> If each is run separately they'll both output, but called in the same cell only the last one seems to show up (unless I changed it to use print).

Having taught our lesson last week, I'm very tempted to propose to create a "Jupyter Gotchas" section in our Instructor Guide to warn instructors of all the gotchas they might face when they teach using Jupyter Notebooks.